### ISSUE TITLE
css-backdrop-filter => change to "developing"?

### ISSUE THREAD

**Issue by scheinercc** [mozilla/platform-status#588]
Shouldn't https://platform-status.mozilla.org/#css-backdrop-filter be set to "developing" as https://bugzilla.mozilla.org/show_bug.cgi?id=1178765 just landed?

**Comment by chrisdavidmills** [comment]
Yeah, this probably makes sense. Fancy submitting a PR for it? You need to change the "firefox_status" field in https://github.com/mozilla/platform-status/blob/master/features/css-backdrop-filter.md to "in-development"
